ggdp/resources.py

```diff
@@ -65,3 +65,15 @@ def upload_csv(self, df: pd.DataFrame, name: str) -> requests.Response:
         response.raise_for_status()
 
         return response
+
+
+class GrantsStackIndexerGraphQL(ConfigurableResource):
+    ENDPOINT: str = "https://grants-stack-indexer-v2.gitcoin.co/graphql"
+
+    def query(self, query: str, variables: dict = {}):
```

Comment:
Is this a good place to add `retry` decorator? 

At the moment if single fetch fails here, so does the parent asset using this resource and so does entire the pipeline.

Reply:
:100: